narwhals/_dask/expr.py

```diff
@@ -69,6 +73,7 @@ def _from_call(
         call: Any,
         expr_name: str,
         *args: Any,
+        is_scalar: bool,
```

Comment:
One tiny and nitty objection on the naming: what about `returns_scalar`? 

- it is already used in the pandas-like API (for a similar reason/behavior)
- Expr itself is _not_ a scalar, but _returns_ one